docs/configuration/indexes.md

```diff
@@ -134,12 +134,12 @@ name = "internal"
 url = "https://example.com/simple"
 ```
 
-From there, you can set the `UV_INDEX_INTERNAL_USERNAME` and `UV_INDEX_INTERNAL_PASSWORD`
+From there, you can set the `UV_HTTP_BASIC_INTERNAL_USERNAME` and `UV_HTTP_BASIC_INTERNAL_PASSWORD`
```

Comment:
@charliermarsh I can't remember which one of these we thought was better. Wasn't it the `UV_INDEX_` prefix?

Reply:
(That's also what's mentioned in the PR summary, so I presume the code is wrong here)

Reply:
Oh I think I meant to do `UV_HTTP_BASIC` but honestly I can't remember

Reply:
I thought we were intentionally deviating from what Poetry does?

Reply:
I don't remember honestly. We can't easily change it now that it's out there though -- it would be breaking.

Reply:
It's possible I might be the only one that's used it successfully though 🤣 
and I don't mind you breaking it on me

Reply:
I guess we could call it a bug since the documentation differs. That seems fine, actually. Ok, lets use `UV_INDEX_INTERNAL_USERNAME` and `UV_INDEX_INTERNAL_PASSWORD`.